### PR TITLE
Remove whitelist and enforce minting limit in contract.

### PR DESCRIPTION
**Pull request by Ramarti**
(Note: This PR goes after #108 )
- Remove whitelist checks from `beforeTokenTransfer()` in `FortaCommon`, plus all related whitelist methods. 
- FORT max supply is now explicit via check in`mint` function (effectively disabling minting in deployed token in Mainnet)
- Remove obsolete scripts (staking during Fortification Phase, whitelist related scripts, etc)

### CODE REVIEW

File: contracts/token/FortaCommon.sol [hunk 1]
```diff
@@ -17,8 +17,6 @@ abstract contract FortaCommon is AccessControlUpgradeable, ERC20VotesUpgradeable
     
     bool public whitelistDisabled; // __gap[50] -> __gap[49]
```
Comment (frangio) [inline contracts/token/FortaCommon.sol:18]: Can we add a comment saying this is deprecated now?

File: contracts/token/Forta.sol [hunk 2]
```diff
@@ -36,7 +40,7 @@ contract Forta is FortaCommon {
      * @return version of FORT deployed in L1
```
Comment (frangio) [inline contracts/token/Forta.sol:40]: We should have better documentation about what this version number is meant to communicate, and instructions for when and how it should be incremented. For example, should every upgrade have a new version? Only if the interface changes? Should it follow semver? Or what does each component of the version represent?

I'm not sure if bumping from 0.1.0 to 0.1.1 here respects the expectations that we've set for the function getter.

**Review by frangio — COMMENTED** [review]
Some minor comments.

Have you considered removing the WHITELIST_ROLE and WHITELISTER_ROLE constants? What about the setRoleAdmin function calls in the initializer? I imagine removing the constants and their public getters may be a stronger breaking change that we might want to avoid.

File: contracts/token/Forta.sol (hunk 2 above)
Comment (Ramarti) [reply contracts/token/Forta.sol:40]: The idea is to follow semver. Fair enough, removing whitelist should be a minor, not a patch.
I'm updating the README with more strict definitions.
I'm noticing also the global package version is set to 1.0.0 for the first deployment, and it should have been 0.1.0 so it is more in synch with the contracts.

File: contracts/token/FortaCommon.sol (hunk 1 above)
Comment (Ramarti) [reply contracts/token/FortaCommon.sol:18]: What about changing the name to something like `deprecated_whitelistDisabled` ?

**Comment by Ramarti** [comment]
> Have you considered removing the WHITELIST_ROLE and WHITELISTER_ROLE constants? What about the setRoleAdmin function calls in the initializer? I imagine removing the constants and their public getters may be a stronger breaking change that we might want to avoid.

I have thought about it. The crux that made me keep all that is this line:

https://github.com/forta-network/forta-contracts/blob/3fdecbf5f79cda53f28eb4be49786d767475fa2b/contracts/vesting/escrow/StakingEscrowFactory.sol#L68

`StakingEscrowFactory` is deployed and immutable, so removing WHITELIST_ROLE and WHITELISTER_ROLE everywhere would remove context for that line. Also in a more abstract way, there _was_ a whitelist and erasing it altogether might seem disingenuous?

The alternative would be to take everything out and redeploy `StakingEscrowFactory`, which implies upgrading 5 or 6 deployed `VestingWalletV2'

Thoughts?

File: contracts/token/FortaCommon.sol (hunk 1 above)
Comment (frangio) [reply contracts/token/FortaCommon.sol:18]: Works for me. We will need to add the following to appease the upgrade validation:

```
/// @custom:oz-renamed-from whitelistDisabled
```

**Comment by frangio** [comment]
I'm ok with keeping the roles, but would add a comment saying they are deprecated and not used.

**Review by frangio — APPROVED** [review]
LGTM

**Pull request merged**